src/css/components/_captions-settings.scss

```diff
@@ -26,30 +26,26 @@
   .vjs-text-track-settings .vjs-modal-dialog-content {
     display: grid;
     grid-template-columns: 1fr 1fr;
-    grid-template-rows: 1fr auto;
+    grid-template-rows: 1fr;
   }
 
-  .vjs-text-track-settings .vjs-track-settings-colors {
-    display: block;
-    grid-column: 1;
-    grid-row: 1;
+  .vjs-text-track-settings .vjs-track-settings-controls {
+    grid-column: 1 / -1;
   }
 
-  .vjs-text-track-settings .vjs-track-settings-font {
-    grid-column: 2;
-    grid-row: 1;
+  // 1 column for small players
+  .vjs-layout-small .vjs-text-track-settings .vjs-modal-dialog-content ,
+  .vjs-layout-x-small .vjs-text-track-settings .vjs-modal-dialog-content,
+  .vjs-layout-tiny .vjs-text-track-settings .vjs-modal-dialog-content {
+    grid-template-columns: 1fr;
   }
 
-  .vjs-text-track-settings .vjs-track-settings-controls {
-    grid-column: 2;
-    grid-row: 2;
-
-  }
 }
 
 // Form elements
 .vjs-track-setting > select {
   margin-right: 5px;
```

Comment:
I think we ought to make this a bit bigger. And `em`s instead - maybe `1em`?

Also, I wonder if we can get some vertical spacing for the `select` controls? One way to do this would be:

- Add `margin-bottom: 1em;` here.
- Remove all `margin` from `.vjs-text-track-settings fieldset`.

That looks a lot better at all sizes, I think.

Reply:
Works for me.

Reply:
testing